javascript/ql/src/semmle/javascript/dataflow/internal/FlowSteps.qll

```diff
@@ -115,6 +115,35 @@ predicate callStep(DataFlow::Node pred, DataFlow::Node succ) {
     argumentPassing(_, pred, _, parm) and
     succ = DataFlow::parameterNode(parm)
   )
+  or
+  constructorReceiverToMethodStep(pred, succ)
+}
+
+/**
+ * Holds if `pred` is the receiver of a class constructor and `succ` is the receiver of one
+ * of the methods of that class.
+ */
+predicate constructorReceiverToMethodStep(DataFlow::Node pred, DataFlow::Node succ) {
+  exists (ClassDefinition class_ |
+    pred = DataFlow::thisNode(class_.getConstructor().getBody()) and
+    succ = DataFlow::thisNode(class_.getAMethod().getBody())
+  )
+  or
+  exists (DataFlow::FunctionNode ctor, DataFlow::SourceNode prototype, DataFlow::PropWrite write |
```

Comment:
This disjunct seems useful in a separate predicate or module for prototype reasoning (later).

Reply:
Again, I believe `ClassNode` resolved this comment